.circleci/config.yml

```
@@ -54,3 +65,24 @@ workflows:
             - lint-changed
             - yarn/type-check
             - yarn/update-cache
+      - yarn/run:
+          name: build-docs
+          script: "deploy-storybook"
+          post-steps:
+            - persist_to_workspace:
+                root: .
+                paths:
+                  - storybook_build
+          filters:
+            branches:
+              only: master
+      - upload-docs:
+          context: static-sites-uploader
+          requires:
+            - build-docs
```

```suggestion
            - build-storybook
```

not really a necessary change, but it'd be a little more clear